mpd_client/src/commands/definitions.rs

```diff
@@ -938,6 +938,189 @@ enum TagTypesAction {
     Enable(Vec<Tag>),
 }
 
+/// `sticker get` command
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct StickerGet {
+    uri: String,
+    name: String,
+}
+
+impl StickerGet {
+    /// Get the sticker `name` for the song at `uri`
+    pub fn new(uri: String, name: String) -> Self {
+        Self { uri, name }
+    }
+}
+
+impl Command for StickerGet {
+    type Response = res::StickerGet;
+
+    fn into_command(self) -> RawCommand {
+        RawCommand::new("sticker")
+            .argument("get")
+            .argument("song")
+            .argument(self.uri)
+            .argument(self.name)
+    }
+}
+
+/// `sticker set` command
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct StickerSet {
+    uri: String,
+    name: String,
+    value: String,
+}
+
+impl StickerSet {
+    /// Set the sticker `name` to `value` for the song at `uri`
+    pub fn new(uri: String, name: String, value: String) -> Self {
+        Self { uri, name, value }
+    }
+}
+
+impl Command for StickerSet {
+    type Response = res::Empty;
+
+    fn into_command(self) -> RawCommand {
+        RawCommand::new("sticker")
+            .argument("set")
+            .argument("song")
+            .argument(self.uri)
+            .argument(self.name)
+            .argument(self.value)
+    }
+}
+
+/// `sticker delete` command
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct StickerDelete {
+    uri: String,
+    name: String,
+}
+
+impl StickerDelete {
+    /// Delete the sticker `name` for the song at `uri`
+    pub fn new(uri: String, name: String) -> Self {
+        Self { uri, name }
+    }
+}
+
+impl Command for StickerDelete {
+    type Response = res::Empty;
+
+    fn into_command(self) -> RawCommand {
+        RawCommand::new("sticker")
+            .argument("delete")
+            .argument("song")
+            .argument(self.uri)
+            .argument(self.name)
+    }
+}
+
+/// `sticker list` command
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct StickerList {
+    uri: String,
+}
+
+impl StickerList {
+    /// Lists all stickers on the song at `uri`
+    pub fn new(uri: String) -> Self {
+        Self { uri }
+    }
+}
+
+impl Command for StickerList {
+    type Response = res::StickerList;
+
+    fn into_command(self) -> RawCommand {
+        RawCommand::new("sticker")
+            .argument("list")
+            .argument("song")
+            .argument(self.uri)
+    }
+}
+
+/// Operator for full (filtered) version
+/// of `sticker find` command
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum StickerFindOperator {
+    /// = operator
+    Equals,
+    /// < operator
+    LessThan,
+    /// > operator
+    GreaterThan,
+}
+
+/// `sticker find` command
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct StickerFind {
+    uri: String,
+    name: String,
+    operator: Option<StickerFindOperator>,
+    value: Option<String>,
```

Comment:
A situation where there is only one of these two values does not exist I think, so this should be collapsed into `Option<(StickerFindOperator, String)>`.